helm/webapp/values-b861c7-dev.yaml

```diff
@@ -12,3 +12,24 @@ sso:
 env:
   IDIR_JWKS_URI: https://dev.oidc.gov.bc.ca/auth/realms/onestopauth/protocol/openid-connect/certs
   IDIR_ISSUER: https://dev.oidc.gov.bc.ca/auth/realms/onestopauth
+
```

Comment:
Oh so this set up lets us pass the postgress and patroni values  in in one place

Reply:
yes, it is, we can even add `db backup` helm chart in the same chart.